sounddevice.py

```diff
@@ -1877,9 +1877,19 @@ def input_underflow(self):
         This can only happen in full-duplex streams (including
         `playrec()`).
 
+        The bit may be set with ``input_underflow = True``, and
+        cleared with ``input_underflow = False``.
```

Comment:
I don't really like this documentation.

I don't really like to repeat this for every field, because I don't think its that important for most people.

More importantly, it's not working, you would have to have something like `mystatus.input_underflow = True`.

What about adding a second example to the already existing example:

https://github.com/spatialaudio/python-sounddevice/blob/5c1a9de051cfa6b72bf26923c37a43c5ee5a0a59/sounddevice.py#L1828-L1840

I don't think we have to repeat this for every single field name, our users should be able to infer that it works for every field, not only the ones that happen to be used in the example.

Reply:
I have removed the previous documentation and replaced it with an example:

    The values may also be set and cleared by the user:                                                                           
                                                                                                                                  
    >>> import sounddevice as sd                                                                                                  
    >>> cf = sd.CallbackFlags()                                                                                                   
    >>> cf                                                                                                                        
    <sounddevice.CallbackFlags: no flags set>                                                                                     
    >>> cf.input_underflow = True                                                                                                 
    >>> cf                                                                                                                        
    <sounddevice.CallbackFlags: input underflow>                                                                                  
    >>> cf.input_underflow = False                                                                                                
    >>> cf                                                                                                                        
    <sounddevice.CallbackFlags: no flags set>